x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/FunctionRegistryTests.java

```diff
@@ -113,7 +113,7 @@ public void testDateTimeFunction() {
         }, "DUMMY_FUNCTION"));
         FunctionDefinition def = r.resolveFunction(ur.name());
         assertEquals(ur.location(), ur.buildResolved(providedConfiguration, def).location());
-        assertTrue(def.datetime());
+        assertTrue(def.extractViable());
```

Comment:
If `extractViable` should be `true` only for `EXTRACT` approach for date/time functions, why is `true` here even for the case https://github.com/elastic/elasticsearch/blob/9798b187509e5071c6eb9add757af7eac504fcbf/x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/FunctionRegistryTests.java#L106 (where `STANDARD` is chosen)? What I'm trying to say is, is `extractViable` a proper name for this boolean or the previous version was more descriptive?

Reply:
I don't follow. The previous name was `datetime` yet it was used to indicate a datetime function used for extract. `NOW` is a datetime function yet cannot be used by `EXTRACT`.
Hence the rename to include `extract` - maybe `extract` is not good enough but `datetime` is a misnomer (which made sense back then when all date time functions were extracting).